src/Core/Components/Switch/FluentSwitch.razor.cs

```diff
@@ -9,7 +9,7 @@
 namespace Microsoft.FluentUI.AspNetCore.Components;
 
 /// <summary>
-/// The FluentSwitch component is used to render a Switch
+/// A FluentSwitch component represents a physical switch that allows someone to choose between two mutually exclusive options.
```

Comment:
Never use 'you', 'the developer', 'someone', etc in docs

/// A FluentSwitch component represents a physical switch that allows a choice between two mutually exclusive options.